<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>868</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KARIN 2.0 (alpha)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="reflectivityTab">
         <attribute name="title">
          <string>Specular</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QStackedWidget" name="SpecularTools">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <layout class="QGridLayout" name="gridLayout_2">
                    <item row="4" column="0">
                     <widget class="QLabel" name="PeriodXray">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                      <property name="text">
                       <string>Period: -- Ã…</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0" alignment="Qt::AlignLeft">
                     <widget class="QListWidget" name="peakList">
                      <property name="sizePolicy">
                       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                       </sizepolicy>
                      </property>
                     </widget>
                    </item>
                    <item row="5" column="0">
                     <layout class="QGridLayout" name="gridLayout">
                      <item row="1" column="0">
                       <widget class="QPushButton" name="removePeak_button">
                        <property name="text">
                         <string>Remove mode</string>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="2">
                       <widget class="QPushButton" name="removeAll_button">
                        <property name="text">
                         <string>Clear all peaks</string>
                        </property>
                       </widget>
                      </item>
                      <item row="0" column="0" alignment="Qt::AlignHCenter">
                       <widget class="QPushButton" name="DetectPeaks_button">
                        <property name="sizePolicy">
                         <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                         </sizepolicy>
                        </property>
                        <property name="minimumSize">
                         <size>
                          <width>112</width>
                          <height>36</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>112</width>
                          <height>36</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Detect Peaks</string>
                        </property>
                       </widget>
                      </item>
                      <item row="1" column="2">
                       <widget class="QPushButton" name="dragMode_button">
                        <property name="text">
                         <string>Drag mode</string>
                        </property>
                        <property name="checkable">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_2">
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_5">
                  <item row="0" column="0">
                   <widget class="QPushButton" name="Insert_line_button">
                    <property name="text">
                     <string>Insert vertical line</string>
                    </property>
                    <property name="checkable">
                     <bool>true</bool>
                    </property>
                    <property name="checked">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="SpecReflectivity_Xray" stretch="0">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetNoConstraint</enum>
              </property>
              <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
               <widget class="QPushButton" name="openSampleDB_button">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Open SampleDB</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="specularTab">
         <attribute name="title">
          <string>Off-specular</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="offSpecReflectivity_Xray"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Simulations</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Calculator</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>952</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_SampleDB"/>
    <addaction name="actionOpen_single_specular_file"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuReflectivity">
    <property name="title">
     <string>Reflectivity</string>
    </property>
    <widget class="QMenu" name="menuShow_2">
     <property name="title">
      <string>View</string>
     </property>
     <addaction name="actionSpecular_X_ray_2"/>
     <addaction name="actionOff_specular_X_ray"/>
     <addaction name="separator"/>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuTools">
     <property name="title">
      <string>Tools</string>
     </property>
     <addaction name="actionDetect_peaks"/>
     <addaction name="actionData_tools"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="menuTools"/>
    <addaction name="menuShow_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuReflectivity"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_SampleDB">
   <property name="text">
    <string>Open SampleDB</string>
   </property>
   <property name="shortcut">
    <string comment="test" extracomment="test"/>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionOff_specular">
   <property name="text">
    <string>Off-specular</string>
   </property>
  </action>
  <action name="actionNeutrons">
   <property name="text">
    <string>Neutrons</string>
   </property>
  </action>
  <action name="actionX_Rays">
   <property name="text">
    <string>X-Rays</string>
   </property>
  </action>
  <action name="actionOpen_single_specular_file">
   <property name="text">
    <string>Open single specular file</string>
   </property>
  </action>
  <action name="actionSpecular_X_ray">
   <property name="text">
    <string>Specular X-ray</string>
   </property>
  </action>
  <action name="actionSpecular_X_ray_2">
   <property name="text">
    <string>Xrays</string>
   </property>
  </action>
  <action name="actionOff_specular_X_ray">
   <property name="text">
    <string>Neutrons</string>
   </property>
  </action>
  <action name="actionSpecular_neutron">
   <property name="text">
    <string>Specular neutron</string>
   </property>
  </action>
  <action name="actionOff_specular_neutron">
   <property name="text">
    <string>Off-specular neutron</string>
   </property>
  </action>
  <action name="actionData_tools">
   <property name="text">
    <string>General analysis</string>
   </property>
  </action>
  <action name="actionDetect_peaks">
   <property name="text">
    <string>Detect peaks</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
